### ISSUE TITLE
Previous and Next buttons are not working any more in the slideshow, in public links

### ISSUE THREAD

**Issue by oparoz** [nextcloud/gallery#270]
### Steps to reproduce

1. Share an album via a link
1. Click on an image
1. Click on one of the arrows (left or right)

### Expected behaviour

Slideshow should go to the next image

### Actual behaviour

Nothing happens.

Probably a regression introduced when adding sharing to the slideshow. There were some CSS changes

### Client configuration

**Browser:**

**Operating system:**

#### Browser log

<details>
<summary>Browser log</summary>

```
Uncaught TypeError: Cannot read property 'hideDropDown' of undefined
    at Controls._next (slideshowcontrols.js?v=d433c9d…-38:353)
    at Controls.<anonymous> (slideshowcontrols.js?v=d433c9d…-38:55)
    at HTMLInputElement.dispatch (core.js:3)
    at HTMLInputElement.r.handle (core.js:3)
```

</details>

**Comment by icewind1991** [comment]
still works fine for me

**Comment by oparoz** [comment]
I tested on c.nc, so maybe it's not running the latest code. I'll try on something else.

**Comment by oparoz** [comment]
Still happening in RC1
https://cloud.nextcloud.com/apps/gallery/s/0WDHAi9p5n0JSew

**Comment by michag86** [comment]
@icewind1991 The problem only occours, if you choose the gallery view. In the file view it is working fine.

Tested on RC3

**Comment by icewind1991** [comment]
For some reason `vendor/nextcloud/share.js` isn't being loaded, but I can't reproduce this on my own instances

**Comment by oparoz** [comment]
Could it be due to the new CSS merger? Maybe it only works in debug mode.

**Comment by icewind1991** [comment]
I also tested it with debug disabled

**Comment by realitygaps** [comment]
I'm also seeing this happening on our instance

**Comment by tribut** [comment]
After upgrading to NC 12.0.0 and updating the gallery app, I'm also seeing this.

**Comment by wronny** [comment]
Very happy that I found this fix #270 here. Works fine on my nextcloud.

**Comment by freechelmi** [comment]
Different problem here : after you delete an image in slideshow mode , navigating with keys does not work anymore
